packages/ascii.rb

```diff
@@ -8,8 +8,16 @@ class Ascii < Package
   source_sha256 'a94bb3970e8f1f63566f055517aecbdd46b11c4ccf142f77ffb80a79994f03a9'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/ascii-3.16-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/ascii-3.16-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/ascii-3.16-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/ascii-3.16-1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '8afd3dd8d8bbb56f039bd51dd2e0a65fdd23449b3858fb328fd0d2cbb75382b9',
+     armv7l: '8afd3dd8d8bbb56f039bd51dd2e0a65fdd23449b3858fb328fd0d2cbb75382b9',
+       i686: '9c5f93b333c1b40e558c76ab082bf5cfd7541a6f36f3c4561250e7dc2db45d26',
+     x86_64: 'cfce0c56efacea56a77d481d6f5001645972fa4780a2a8dea921f9435dffc466',
```

Comment:
Don't we want this package updated to version 3.18?

Reply:
arg....should be...